### DISCUSSION TITLE
有关 THU 选课的一些常见问题与注意事项

### DISCUSSION THREAD

**Discussion by giscus** [ouuan/iles-blog#10] [Comments]
# 有关 THU 选课的一些常见问题与注意事项

还记得刚入学的时候，我一开始完全没弄明白要怎么选课，当时就想着弄明白了要写一篇选课指南。
后来，我弄明白之后发现不记得自己以前为什么不明白了，写不出来了（
再后来，我发现一些身边的同学到了选大二的课的时候还没弄明白选课流程（
所以，虽然选课指南是写不出来了，姑且是列举一些常见问题与注意事项，供参考。

https://ouuan.moe/post/2022/07/thu-course-selection-tips

**Comment by Chiyuru** [comment]
我记得我一开始最搞不清楚的问题就是怎么手选&怎么替代，所以放弃了用数分高代替代微 A 线代，血亏（
可以补个链接https://webvpn.tsinghua.edu.cn/http/77726476706e69737468656265737421fae0429e207e6b597d469dbf915b243d8ae9128e1cdcffb247/download.jsp?attachSeq=3676 ，手选的话只要还有课余量基本就能选上，不放心可以提前给老师发个邮件，没什么特殊的要求（
另外手选的时候还可以冲突选课（就是可以选和已有课程时间冲突的课），有点冷知识（
